challenge_one/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "nethermind-bot-deploy-and-update-monitor",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
+  "chainIds": [
+    137
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "alert": "npm run build && forta-agent run --alert",
+    "sequence": "npm run build && forta-agent run --sequence",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "info": "forta-agent info",
+    "logs": "forta-agent logs",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest",
+    "format": "prettier --write \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "big-number": "^2.0.0",
+    "forta-agent": "^0.1.16",
+    "forta-agent-tools": "^3.2.3",
```

Comment:
In this bot, `forta-agent-tools` is only used in testing, so it should be moved to `devDependencies`.